src/app/applications/html/_app-directive.html

```diff
@@ -10,7 +10,7 @@
 	</p>
 	
 	<p class="app-avatar">
-		<img ng-src="{{ findIcon( app )}}" alt="application's icon" ng-class="findAvatar( app )" />
+ 		<img id="img-id" ng-src="{{ findIcon( app )}}" alt="application's icon" ng-class="findAvatar( app )" title="Click to replace this image" ng-click="selectFile()"/>
```

Comment:
The title attribute should be used only for the overview page.
So, it should be wrapped within a **ng-if** tag.